src/sasjs-request/index.js

```diff
@@ -8,7 +8,7 @@ import {
   createFolder
 } from '../utils/file-utils'
 import { getVariable } from '../utils/utils'
-import { getAccessToken } from '../utils/auth-utils'
+import { getAccessToken } from '../utils/config-utils'
```

Comment:
The method signatures are different, so this won't work without changing the call below.